pkg/replication/replicator.go

```diff
@@ -14,6 +16,10 @@ import (
 
 const DefaultConcurrentReality = 3
 
+// Normal replications will have no timeout, but daemon sets will
+// because it is unlikely that all hosts are healthy at all times
```

Comment:
i'm not sure I understand the justification, shouldn't a daemon set keep trying forever to sync out the version it is configured to sync?